suggestion/src/util/hashIngredientList.ts

```diff
@@ -0,0 +1,11 @@
+export function hashIngredientList(recipeHash: number[]): string {
```

Comment:
Is this in a different PR too?

Reply:
yeah, I needed this function to write mine. Dw about it, we are gonna clean it, integrate them and try to test them on Saturday.